d2l-scroll-wrapper.html

```diff
@@ -125,6 +125,7 @@
 
 			attached: function() {
 				this.async(this.checkScrollbar);
+				this.async(this.checkScrollbar);
```

Comment:
I'd at least throw a comment in here explaining why it's getting the duplicate call, just so someone doesn't remove it in the future thinking it was a mistake.